src/ex1/script.js

```diff
@@ -0,0 +1,284 @@
+//Selectors
+const todoInput = document.getElementById("input-txt");
+const todoButton = document.getElementById("add-btn");
+const todoList = document.getElementById("list-element");
+const clearAllBtn = document.getElementById("clearAll-btn");
+const sortBtn = document.getElementById("sort-btn");
+const inputBox = document.querySelector(".inputField input");
+const svgContainer = document.getElementById("svg");
+const todoElement = document.getElementById("list-element");
+const footerSpan = document.querySelector(" .footer span ");
+
+const animItem = bodymovin.loadAnimation({
+  wrapper: svgContainer,
+  animType: "svg",
+  loop: false,
+  autoplay: false,
+  path: "https://assets4.lottiefiles.com/private_files/lf30_5aubt2fy.json",
+});
+
+const pendingTasksCount = document.querySelector(".pendingTasksCount");
+const ENTER_KEY = 13;
+const TRASH_ANIMATION_TIMEOUT = 500;
+
+//Event Listeners
+document.addEventListener("DOMContentLoaded", getTodos);
+todoButton.addEventListener("click", addTodo);
+todoInput.addEventListener("keyup", todoEnter);
+todoList.addEventListener("click", deleteTask);
+sortBtn.addEventListener("click", sortTasks);
+clearAllBtn.addEventListener("click", clearAll);
+todoElement.addEventListener("click", alertTask);
+
+//Functions
+
+//add alert feature when task is clicked
+function alertTask(e) {
+  const item = e.target;
+
+  if (item.classList[0] === "todo") {
+    const todo = item.parentElement;
+
+    let message = "📝" + item.innerText;
+    return alert(item.innerText);
+    /*will be used later in the project launch_listElement(message); */
+  }
+}
+
+//deactive / active buttons when todos are added / cleared
+function listStyling(todosCount) {
+  if (!todosCount) {
+    clearAllBtn.classList.add("deactive");
+    sortBtn.classList.add("deactive");
+    footerSpan.classList.add("deactive");
+  } else {
+    clearAllBtn.classList.remove("deactive");
+    sortBtn.classList.remove("deactive");
+    footerSpan.classList.remove("deactive");
+  }
+}
+
+//todo input styling
+todoInput.onkeydown = () => {
+  if (todoInput.value.length) {
+    inputBox.classList.add("inactive");
+    todoButton.classList.add("inactive");
+  }
+
+  if (!todoInput.value.length && !todoList.childElementCount) {
+    inputBox.classList.remove("inactive");
+    todoButton.classList.remove("inactive");
+  }
+};
+
+//add todo to the todo list
+function addTodo(event) {
+  if (!todoInput.value.trim()) {
+    return launch_toast();
```

Comment:
Very good, using the false condition to break out of the function if condition not met first.